Options.py

```diff
@@ -1229,7 +1229,7 @@ class CommonOptions(metaclass=OptionsMetaProperty):
     progression_balancing: ProgressionBalancing
     accessibility: Accessibility
 
-    def as_dict(self, *option_names: str, casing: str = "snake") -> typing.Dict[str, typing.Any]:
+    def as_dict(self, *option_names: str, casing: str = "snake", toggles_as_bools: bool = False) -> typing.Dict[str, typing.Any]:
```

Comment:
I think this makes this line more than 120 characters.

Reply:
```suggestion
    def as_dict(self,
                *option_names: str,
                casing: typing.Literal["snake", "camel", "pascal", "kebab"] = "snake",
                toggles_as_bools: bool = False) -> typing.Dict[str, typing.Any]:
```
This is how I would do it to make room for the improved `casing` typing that isn't relevant to this PR...